content/customize-gnome.md

```diff
@@ -18,117 +18,210 @@ section: software
 
-![backslide](/images/customize-gnome/backslide.png)
+</details>
 
-This extension is a wallpaper slideshow for your desktop.  After choosing a folder of pictures, they will be rotated automatically at a predefined time period.
+### [Cosmic Workspaces](https://github.com/pop-os/cosmic-workspaces)
 
----
+<details>
+  <summary>Click to expand</summary>
 
-## Window List
+A fork of this [extension](https://github.com/RensAlthuis/vertical-overview) designed to return the GNOME workspaces to a vertical layout rather than the horizontal view introduced in GNOME 40.
 
-[Window List](https://extensions.gnome.org/extension/602/window-list/)
+Workspace behaviors can be edited in Settings.
 
-![window-list](/images/customize-gnome/window-list.png)
+![workspaces-and-settings](/images/customize-gnome/workspaces-and-settings.png)
 
-This extension adds a window list to the bottom of the monitors.  This shows what windows are open, and allows for them to be minimized, maximized, and closed from the bar, similar to Microsoft Window's window list.
+</details>
 
----
+### [Desktop Icons NG (DING)](https://gitlab.com/rastersoft/desktop-icons-ng)
 
-## Gnomenu
+<details>
+  <summary>Click to expand</summary>
 
-[Gnomenu](https://extensions.gnome.org/extension/608/gnomenu/)
+Since Desktop icons have been deprecated in mainline GNOME, Pop!_OS includes a GNOME extension to bring this feature back.
 
-![gnomenu](/images/customize-gnome/gnomenu.png)
+</details>
 
-This extension adds a traditionally styled, full featured menu to GNOME.  This menu is similar to a traditional Window's Start Menu, and is highly customizable for how it looks and functions.
+### [Pop COSMIC](https://github.com/pop-os/cosmic)
 
----
+<details>
+  <summary>Click to expand</summary>
 
-## Appindicator Support and TopIcons Plus
+![cosmic-dock](/images/customize-gnome/cosmic-dock.png)
 
-[Appindicator Support](https://extensions.gnome.org/extension/615/appindicator-support/)
-[Top Icons](https://extensions.gnome.org/extension/1031/topicons/)
+Computer Operating System Main Interface Components **(C.O.S.M.I.C.)**
 
-![topicons](/images/customize-gnome/topicons.png)
+COSMIC is the name for the main shell components in Pop_Shell (separate from the tiling and window-management components). It makes the following changes to the standard GNOME Shell environment:
 
-Appindicator Support and TopIcons Plus both serve to add the ability for programs to use an indicator icon to show they are running.  This functionality has been removed in recent versions of GNOME, and an extension is needed for the functionality.  These 2 programs work slightly differently, and work better in some setups vs. others.  If one doesn't work, try the other one.
+- Dock
+- Multi-Monitor improvements
+- Configuration options
+- Separated Workspaces overview from Applications.
```

Comment:
Structure suggestion - If I understand correctly, COSMIC encompasses several components listed in this article. It may be helpful to include this higher up, and link to other articles or relevant sections in the article:

- Dock (link to Dock section)
- Multi-monitor improvements (link to HiDPI Daemon article)
- Configuration options (this one is a little vague, might be useful to elaborate)
- Separated Workspaces overview from Applications (I'm not 100% sure what this means, this is also the only list item with a period, and I'm not sure it needs one)

Reply:
I don't think anything was updated in HiDPI daemon for C.O.S.M.I.C. 
I was referring to things like improved Pop_Shell behavior, new navigation methods etc.